contributed_definitions/NXoptical_spectroscopy.nxdl.xml

```diff
@@ -400,6 +404,9 @@ TODO:
                         <item value="halogen lamp"/>
                         <item value="LED"/>
```

Comment:
lowercase

Reply:
I would stick to uppercase notation for acronyms, as here: https://github.com/nexusformat/definitions/blob/main/contributed_definitions/NXmpes.nxdl.xml#L140-L145

Reply:
Or is there a specific reason? "led" just looks wierd compared to "LED".

Reply:
resolved as lowercase notation is not mandatory